connector/connect/src/main/protobuf/spark/connect/relations.proto

```diff
@@ -123,8 +123,16 @@ message Union {
 // Relation of type [[Fetch]] that is used to read `limit` / `offset` rows from the input relation.
 message Fetch {
```

Comment:
qq why is this Fetch? There are two separate logical plans `Limit` and `Offset`.